src/main/java/mekanism/tools/item/ItemMekanismPaxel.java

```diff
@@ -1,25 +1,98 @@
 package mekanism.tools.item;
 
 import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
 import javax.annotation.Nonnull;
-import mekanism.tools.common.ToolUtils;
+import mekanism.common.util.LangUtils;
+import mekanism.tools.common.IHasRepairType;
+import mekanism.tools.common.Materials;
+import net.minecraft.block.Block;
+import net.minecraft.block.material.Material;
 import net.minecraft.block.state.IBlockState;
-import net.minecraft.init.Blocks;
+import net.minecraft.client.util.ITooltipFlag;
+import net.minecraft.item.ItemAxe;
+import net.minecraft.item.ItemPickaxe;
+import net.minecraft.item.ItemSpade;
 import net.minecraft.item.ItemStack;
+import net.minecraft.item.ItemTool;
+import net.minecraft.world.World;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class ItemMekanismPaxel extends ItemMekanismTool {
+public class ItemMekanismPaxel extends ItemTool implements IHasRepairType {
 
-    public ItemMekanismPaxel(ToolMaterial toolMaterial) {
-        super(4, -2.4F, toolMaterial, new HashSet<>());
+    private static final Set<Block> EFFECTIVE_ON = new HashSet<>();
+
+    static {
+        EFFECTIVE_ON.addAll(ItemPickaxe.EFFECTIVE_ON);
```

Comment:
Is this this (or the AT entries) necessary? - the only method the field is used in has an override in this class...

Reply:
After looking into things a bit more this and the unregistered items are not actually necessary because of forge hooks, and I can remove them if you think I should remove it I can. They just ensure it keeps the same behavior as vanilla if someone directly calls any of the methods.

Reply:
yeah should be enough to rely on forge hooks

Reply:
Left in the one AT addition that is needed so that we don't have to store the value ourselves. (Changing the ItemSword material to protected from private)